src/components/templates/PosterHall/PosterHall.tsx

```diff
@@ -27,14 +27,15 @@ export const PosterHall: React.FC<PosterHallProps> = ({ venue }) => {
 
     searchInputValue,
     setSearchInputValue,
-    liveFilter,
-    setLiveFilter,
   } = usePosters(venue.id);
 
   const shouldShowMorePosters = isPostersLoaded && hasHiddenPosters;
 
   const renderedPosterPreviews = useMemo(() => {
-    return posterVenues.map((posterVenue) => (
+    const sortedVenues = [...posterVenues].sort(
+      (a, b) => (b.recentUserCount ?? 0) - (a.recentUserCount ?? 0)
```

Comment:
Would be useful to have more data to check this one in practice i.e. would it be too often and too disruptive to UX?

Reply:
Agreed. I think this is something that might get changed in the near future when we've seen people actually using it.

Although the counts are only updated once every minute so they're not going to be constantly changing.